packages/ppx_deriving_yojson/ppx_deriving_yojson.3.4/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "whitequark <whitequark@whitequark.org>"
+authors: [ "whitequark <whitequark@whitequark.org>" ]
+license: "MIT"
+homepage: "https://github.com/whitequark/ppx_deriving_yojson"
+doc: "http://whitequark.github.io/ppx_deriving_yojson"
+bug-reports: "https://github.com/whitequark/ppx_deriving_yojson/issues"
+dev-repo: "git://github.com/whitequark/ppx_deriving_yojson.git"
+tags: [ "syntax" "json" ]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "yojson" {< "1.6.0"}
```

Comment:
Should this be `{>= "1.6.0"}`, since it uses the `Yojson.Safe.t` introduced in yojson 1.6.0?
(See https://github.com/ocaml-ppx/ppx_deriving_yojson/pull/90)

Reply:
Yes, of course, thanks! I'll rebase the PR.